honeybadgermpc/hbavss.py

```diff
@@ -299,24 +324,19 @@ def _get_dealer_msg(self, values):
         logger.debug("[%d] Starting Batch AVSS. Id: %s, Dealer Id: %d, Client Mode: %s",
                      self.my_id, avss_id, dealer_id, client_mode)
 
-        broadcast_msg = None if self.my_id != dealer_id else self._get_dealer_msg(values)
+        dispersal_msg_list = None if self.my_id != dealer_id else self._get_dealer_msg(
```

Comment:
nit: Make this into an `if` `else` statement instead of inline `if` since this has become too long.

Reply:
Resolved.